packaging/conda/README.md

```diff
@@ -12,12 +12,7 @@ You'll need `conda-build` installed in your system.
 export KHIOPS_RAW_VERSION=$(scripts/khiops-version)
 export KHIOPS_VERSION=$(echo $KHIOPS_RAW_VERSION | sed 's/-//')
 
-# Windows
 conda build --output-folder ./khiops-conda packaging/conda
-
-# Linux/macOS
-# Note: We need the conda-forge channel to obtain the pinned versions of MPICH
-conda build --channel conda-forge --output-folder ./khiops-conda packaging/conda
```

Comment:
We need to make sure that Conda is configured to use `conda-forge` as its default channel and that the default channel is removed, e.g. by writing:
```
$ conda config --add channels conda-forge
$ conda config --remove channels defaults
```

this would update the `$HOME/.condarc` file.

In the CI this is ensured by the usage of `miniforge-version` in the invocation of the `setup-miniconda` action.

Reply:
Add your comment in the README.md